src/navsat_transform.cpp

```diff
@@ -93,19 +96,45 @@ NavSatTransform::NavSatTransform(const rclcpp::NodeOptions & options)
   // Load the parameters we need
   magnetic_declination_ = this->declare_parameter("magnetic_declination_radians", 0.0);
   yaw_offset_ = this->declare_parameter("yaw_offset", 0.0);
-  broadcast_utm_transform_ = this->declare_parameter("broadcast_utm_transform", false);
-  broadcast_utm_transform_as_parent_frame_ = this->declare_parameter(
-    "broadcast_utm_transform_as_parent_frame", false);
   zero_altitude_ = this->declare_parameter("zero_altitude", false);
   publish_gps_ = this->declare_parameter("publish_filtered_gps", true);
   use_odometry_yaw_ = this->declare_parameter("use_odometry_yaw", false);
   use_manual_datum_ = this->declare_parameter("wait_for_datum", false);
+  use_local_cartesian_ = this->declare_parameter("use_local_cartesian", false);
   frequency = this->declare_parameter("frequency", frequency);
   delay = this->declare_parameter("delay", delay);
   transform_timeout = this->declare_parameter("transform_timeout", transform_timeout);
 
   transform_timeout_ = tf2::durationFromSec(transform_timeout);
 
+  broadcast_cartesian_transform_ =
+    this->declare_parameter("broadcast_utm_transform", broadcast_cartesian_transform_);
+
+  if (broadcast_cartesian_transform_) {
+    RCLCPP_WARN(
+      this->get_logger(), "Parameter 'broadcast_utm_transform' has been deprecated. "
+      "Please use 'broadcast_cartesian_transform' instead.");
+  } else {
+    broadcast_cartesian_transform_ =
+      this->declare_parameter("broadcast_utm_transform", broadcast_cartesian_transform_);
```

Comment:
Shouldn't this be `broadcast_cartesian_transform` then?

Reply:
Yep, thanks.